api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java

```diff
@@ -283,81 +299,78 @@ public void setUp() {
             .participantId(2L);
 
     participantCohortStatus1 =
-        new DbParticipantCohortStatus()
-            .statusEnum(CohortStatus.NOT_REVIEWED)
-            .participantKey(key1)
-            .genderConceptId(TestDemo.MALE.getConceptId())
-            .gender(TestDemo.MALE.getName())
-            .raceConceptId(TestDemo.ASIAN.getConceptId())
-            .race(TestDemo.ASIAN.getName())
-            .ethnicityConceptId(TestDemo.NOT_HISPANIC.getConceptId())
-            .ethnicity(TestDemo.NOT_HISPANIC.getName())
-            .birthDate(new java.sql.Date(today.getTime()))
-            .deceased(false);
+        participantCohortStatusDao.save(
+            new DbParticipantCohortStatus()
+                .statusEnum(CohortStatus.NOT_REVIEWED)
+                .participantKey(key1)
+                .genderConceptId(TestDemo.MALE.getConceptId())
```

Comment:
Maybe rename TestDemo to `TestConcepts`.

Reply:
Done.